Sources/SwifterAuth.swift

```diff
@@ -114,6 +115,63 @@ public extension Swifter {
             }
         }, failure: failure)
     }
+  
+    func authorizeSSO(success: SSOTokenSuccessHandler?,
+                      failure: FailureHandler? = nil) {
```

Comment:
nit: don't need to add a newline if both arguments are short enough for one line.

Reply:
Fixed!
https://github.com/mattdonnelly/Swifter/pull/271/commits/463a6e5253fdedfba01c394c7eb18041a26e6016